.github/workflows/build-images-action.yml

```diff
@@ -0,0 +1,15 @@
+name: build-images-action
+on: [push]
```

Comment:
I don't think we should build all branches! This should be only for `main`

Reply:
Good point. I added that.